.storybook/addons/theme-selector/ThemeSelector.tsx

```diff
@@ -18,14 +18,16 @@ import {
 import { ADDON_EVENT, ADDON_ID } from "./constants";
 
 const ThemeSelector = ({ api }) => {
-  const managerStyles = getManagerStyles();
   const themesList = getThemesList(themes);
   const initialTheme = getInitialTheme(themesList);
   const [selectedTheme, setSelectedTheme] = useState(initialTheme);
 
-  const switchMode = (mode: string) => {
-    const currentTheme = getLocalTheme();
+  const currentTheme = getLocalTheme();
```

Comment:
LGTM 👌🏻 Just one small thing:

I think we should switch this code to the way it previously was because:
- `getManagerStyles` doesn't need the parameter
- and if we switch the docs color mode, the wrong theme (ds3/ds5) is selected on the theme selector sometimes because it's using and old stored value:

https://github.com/lumada-design/hv-uikit-react/assets/43220251/f695d7f7-d00f-4955-9ab3-f19d22772394

Reply:
I've reverted this and a few more things 👍